app/modules/asset/service.server.ts

```diff
@@ -1632,6 +1626,41 @@ export async function createLocationChangeNote({
   }
 }
 
+function getLocationUpdateNoteContent({
```

Comment:
I think we might have to consider doing some restructuring about how our files are managed. Like for example this is not a service function. It doesnt access the DB itself or anything like this. Its more of a utility function.
What do you think about the idea of creating a `utils.server.ts` file inside each module folder, where we place those file? Now we have a general utils folder but that seems a bit messy and not so organized.

Reply:
Yeah this makes more sense than putting everything in `service.server.ts` file